### PR TITLE
Adding bankfeed api

### PR DESCRIPTION
**Pull request by siddharth1729**
I have implemented the api as discussed in my own faishon do let me know if it need to be chged and it folow all norms ples mege this 

Tannks

### CODE REVIEW

File: modules/bankfeed/data/bandata.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const BankData = {
```
Comment (ageddesi) [inline modules/bankfeed/data/bandata.ts:1]: File is spelt wrong should be bankdata.ts

File: modules/bankfeed/data/bandata.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  id: '1234',
+  accountnumber: '1111222233334444',
+  ifsc: 'XYZN01234',
+  sortcode: '1234',
+  isbn: 'dkjdsj',
+  statement: 'This is a bank statemnet',
```
Comment (ageddesi) [inline modules/bankfeed/data/bandata.ts:7]: Statement needs to be an array of transactions.

```
date: TimeStamp
description: String,
amount: number
```

File: modules/bankfeed/api/bankfeed.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import BankData from '../data/bandata';
+
+module.exports = function (app: core.Express) {
+  // Get a list of bank statement
+  app.get('/', (req: Request, res: Response) => {
```
Comment (ageddesi) [inline modules/bankfeed/api/bankfeed.ts:7]: please can this be '/bank/feed/:qty'

qty being the amount of transactions they would like in their statement array. We already have a util to get the qty from the request. getQtyFromRequest

File: modules/bankfeed/api/bankfeed.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    const response = { message: '', data: {} };
+    try {
+      if (req.method === 'get') {
```
Comment (ageddesi) [inline modules/bankfeed/api/bankfeed.ts:10]: For success we like to just use res.json directly and res.status(400) for errors. 

See names-routes.ts for examples.

File: modules/bankfeed/api/bankfeed.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import BankData from '../data/bandata';
+
```
Comment (ageddesi) [inline modules/bankfeed/api/bankfeed.ts:4]: This file will need to be imported into app.ts and will need swagger.json updating with your endpoint data.

**Review by ageddesi — CHANGES_REQUESTED** [review]
Thank you for the work so far. There is some changes that are needed to fall into how we code on the project. All the best. Aaron

File: modules/bankfeed/api/bankfeed.ts (hunk 5 above)
Comment (siddharth1729) [reply modules/bankfeed/api/bankfeed.ts:4]: @ageddesi 
 This part i am not able to understand, although i have implemented all again as you told me pleas have a look

Comment (ageddesi) [reply modules/bankfeed/api/bankfeed.ts:5]: Thanks for the changes, if you look in app.ts you will see how the other route files are imported. If you can do this for this endpoint as well.

Do you also know how to update the swagger.json file with your new endpoints as well?

Comment (siddharth1729) [reply modules/bankfeed/api/bankfeed.ts:5]: No i do not have any idea about this.
I have tried what i could under from app.ts file regarding importing the modules do let me know if any other chnges

Comment (siddharth1729) [reply modules/bankfeed/api/bankfeed.ts:6]: @ageddesi 
Hii if my commit is fine enough then pleas do merge or let me know the chnges which i have to made 
And also Please give some more issues on which i can work for this hactober

**Comment by ageddesi** [comment]
@all-contributors please add @siddharth1729  for code

Thank you @siddharth1729 for the contribution, Happy Hacktoberfest

**Pull request merged**